CHANGELOG.next.asciidoc

```diff
@@ -47,6 +47,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- make `system/filesystem` code sensitive to `hostfs` and migrate libraries to `elastic-agent-opts` {pull}31001[31001]
```

Comment:
Needs fix.

Reply:
what exactly needs to be fixed? is it because it got another changelog line about this comment?

Reply:
This is not part of 7.17